### ISSUE TITLE
Nonfunctional Discourse Link

### ISSUE THREAD

**Issue by infinitymdm** [PySpice-org/PySpice#344]
Links to https://pyspice.discourse.group return a DNS error. I'm trying to find a place to ask a few questions, so it would be pretty great if those were working 😁

On Chromium, I get "DNS_PROBE_FINISHED_NXDOMAIN".

**Comment by daquintero** [comment]
Yeah I got the same, I think maybe we need to update the project as it's quite valuable to mantain it in active development and create another set of infrastructure and community.
